doc/rtd/howto/run_cloud_init_locally.rst

```diff
@@ -70,6 +70,12 @@ Download an Ubuntu image to run:
 
     wget https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img
 
+.. note::
```

Comment:
Since this is already in a note, the "Please be aware" seems redundant.

> is written for x86. 

Correct, but this seems like unnecessary information.

Perhaps:

> This example uses emulated CPU instructions on non-x86 hosts, so it may be slow. To make it faster on non-x86 architectures, one can change the image type and qemu-system-<arch> command name to match the architecture of your host machine.

Reply:
pulled in a laid down this note suggestion for inclusion prior to merge.